apis/management.cattle.io/v3/logging_types.go

```diff
@@ -94,19 +94,27 @@ type LoggingCondition struct {
 }
 
 type ElasticsearchConfig struct {
-	Endpoint     string `json:"endpoint,omitempty" norman:"required"`
-	IndexPrefix  string `json:"indexPrefix,omitempty" norman:"required"`
-	DateFormat   string `json:"dateFormat,omitempty" norman:"required,type=enum,options=YYYY-MM-DD|YYYY-MM|YYYY,default=YYYY-MM-DD"`
-	AuthUserName string `json:"authUsername,omitempty"` //secret
-	AuthPassword string `json:"authPassword,omitempty"` //secret
-	SSLVerify    bool   `json:"sslVerify,omitempty" norman:"required,default=true"`
+	Endpoint      string `json:"endpoint,omitempty" norman:"required"`
+	IndexPrefix   string `json:"indexPrefix,omitempty" norman:"required"`
+	DateFormat    string `json:"dateFormat,omitempty" norman:"required,type=enum,options=YYYY-MM-DD|YYYY-MM|YYYY,default=YYYY-MM-DD"`
+	AuthUserName  string `json:"authUsername,omitempty"` //secret
+	AuthPassword  string `json:"authPassword,omitempty"` //secret
+	Certificate   string `json:"certificate"`
+	ClientCert    string `json:"clientCert"`
+	ClientKey     string `json:"clientKey"`
+	ClientKeyPass string `json:"clientKeyPass"`
+	SSLVerify     bool   `json:"sslVerify" norman:"default=false"`
```

Comment:
default is always false for the boolean, so there is no need to define the norman tag